app/models/general_info.rb

```diff
@@ -44,9 +43,20 @@ class GeneralInfo
             numericality: { only_integer: true, greater_than: 0, less_than_or_equal_to: 20,
                             message: "should be a positive number (1-20)" }
   validates :ori, ori: true
+  validates :address, presence: true
+  validate :incident_unique?
```

Comment:
@AlexNisnevich i love ruby on rails, their principle of "convention over configuration" makes it so much fun to develop applications.